scripts/helper.js

```diff
@@ -0,0 +1,45 @@
+const fs = require("fs");
```

Comment:
To keep things consistant you should use `shelljs` like the other scripts

Reply:
I wouldn't create an "helper" scripts it's too generic. Instead I would create an invidual file per helper like the other helpers. With shelljs I don't even think you really need to define helpers for `clearDir`, `checkFolderExist` etc...